library/system/src/lib/yast2/execute.rb

```diff
@@ -0,0 +1,60 @@
+# ***************************************************************************
+#
+# Copyright (c) 2015 SUSE LLC
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of version 2 of the GNU General Public License as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.   See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, contact Novell, Inc.
+#
+# To contact Novell about this file by physical or electronic mail,
+# you may find current contact information at www.novell.com
+#
+# ***************************************************************************
+
+require "yast"
+require "cheetah"
+
+Yast.import "Installation"
+
+module Yast2
```

Comment:
I see. Hmm, as soon as we step outside the directory, consistency is gone (13.2) : 
```console
$ cd /usr/share/YaST2/lib; grep -rT ^module .| while read F T; do printf '%-50s  %s\n' "$F" "$T"; done
./services-manager/ui_elements.rb                  :module UIElements
./services-manager/services_manager_profile.rb     :module Yast
./installation/clone_settings.rb                   :module Installation
./installation/minimal_installation.rb             :module Installation
./installation/prep_shrink.rb                      :module Installation
./installation/cio_ignore.rb                       :module Installation
./installation/remote_finish_client.rb             :module Installation
./ydocker/run_image_dialog.rb                      :module YDocker
./ydocker/commit_dialog.rb                         :module YDocker
./ydocker/changes_dialog.rb                        :module YDocker
./ydocker/inject_shell_dialog.rb                   :module YDocker
./ydocker/main_dialog.rb                           :module YDocker
./fonts/fpl-add-dialog.rb                          :module FontsConfig
./fonts/select-ebl-dialog.rb                       :module FontsConfig
./fonts/fonts-dialog.rb                            :module FontsConfig
./fonts/shell-commands.rb                          :module FontsConfig
./fonts/fonts-config-state.rb                      :module FontsConfig
./fonts/rich-text-dialog.rb                        :module FontsConfig
./bootloader/boot_record_backup.rb                 :module Bootloader
./bootloader/grub2base.rb                          :module Yast
./bootloader/device_mapping.rb                     :module Bootloader
./yast2/systemd_unit.rb                            :module Yast
./yast2/hw_detection.rb                            :module Yast2
./yast2/systemctl.rb                               :module Yast
./network/edit_nic_name.rb                         :module Yast
./network/install_inf_convertor.rb                 :module Yast
./network/confirm_virt_proposal.rb                 :module Yast
./network/network_autoconfiguration.rb             :module Yast
```

Apparently some packages start with the package name at top namespace level while others wrap it in a common `Yast` module, and yast2.rpm uses `Yast2` as the package name.

I think it would be best to use wrapper `Yast` everywhere, including here.